src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-eks-add-on.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: New Relic Kubernetes Operator from AWS Marketplace
+tags:
+  - Infrastructure
+  - Install the EKS Add-on
+  - NewRelic Operator
+  - AWS EKS Add-on
+metaDescription: New Relic's EKS Add For EKS Monitoring.
```

Comment:
should be "...New Relic's EKS Add-On..."

Reply:
@glennthomas1, addressed, I made the suggested change.